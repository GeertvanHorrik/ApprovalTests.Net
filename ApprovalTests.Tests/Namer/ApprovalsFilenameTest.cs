using ApprovalTests.Namers;
using NUnit.Framework;

namespace ApprovalTests.Tests.Namer
{
    public class ApprovalsFilenameTest
    {
        [Test]
        public void TestMachineSpecificName()
        {
            var approvalsFilename = ApprovalsFilename.Parse(@"C:\Users\olgica\Documents\GitHub\ApprovalTests.Net\ApprovalTests.Tests\Email\EmailTest.Testname.Microsoft_Windows_10_Education.approved.eml");
            Approvals.Verify(approvalsFilename);
            Assert.True(approvalsFilename.IsMachineSpecific);
        }

        [Test]
        public void TestNonMachineSpecificName()
        {
            Approvals.Verify(ApprovalsFilename.Parse(@"C:\Users\olgica\Documents\GitHub\ApprovalTests.Net\ApprovalTests.Tests\Email\EmailTest.Testname.approved.eml"));
        }
    }
}