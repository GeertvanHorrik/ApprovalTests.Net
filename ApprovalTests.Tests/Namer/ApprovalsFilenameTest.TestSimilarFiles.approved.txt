

EmailTest.Testname.Mac.approved.eml
EmailTest.Testname.Microsoft_Windows_10_Education.approved.eml
EmailTest.Testname.Microsoft_Windows_10_Home_N.approved.eml
EmailTest.Testname.Microsoft_Windows_10_Pro.approved.eml
EmailTest.Testname.Microsoft_Windows_Server_2016_Datacenter.approved.eml
EmailTest.Testname.Windows_7.approved.eml
EmailTest.Testname.Windows_8.approved.eml
